catalogd/cmd/catalogd/main.go

```diff
@@ -392,3 +396,16 @@ func podNamespace() string {
 	}
 	return string(namespace)
 }
+
+func clearCacheDir(cacheDirPath string) error {
```

Comment:
May as well rename this to "ensureEmptyDirectory" and throw the `os.MkdirAll` call in there as well?